### PR TITLE
chore(introduction): update Jackson notice in license books

### PR DESCRIPTION
**Pull request by ThorbenLindhauer**
related to camunda/camunda-bpm-platform#3268

### CODE REVIEW

File: content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md [hunk 1]
```diff
@@ -13,25 +13,253 @@ menu:
...
+been in development since 2007.
+It is currently developed by a community of developers.
+## Licensing
+Jackson 2.x core and extension components are licensed under Apache License 2.0
+To find the details that apply to this artifact see the accompanying LICENSE file.
+## Credits
+A list of contributors may be found from CREDITS(-2.x) file, which is included
+in some artifacts (usually source distributions); but is always available
+from the source code management (SCM) system project uses.
+Copyright (c) various contributors: https://github.com/qos-ch/logback/graphs/contributors</pre></details></div>
```
Comment (yanavasileva) [inline content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:28]: ❓ The logback pre is changed by mistake, isn't it?
```suggestion
<div><details><summary>logback-core@1.2.11(<a href="https://opensource.org/licenses/EPL-1.0">EPL 1.0</a>)</summary><pre>Copyright (c) various contributors: https://github.com/qos-ch/logback/graphs/contributors</pre></details></div>
```

Comment (ThorbenLindhauer) [reply content/introduction/third-party-libraries/camunda-bpm-platform-license-book.md:28]: Correct, thanks for raising this.

File: content/introduction/third-party-libraries/camunda-bpm-rpa-bridge-license-book.md
Comment (yanavasileva) [inline content/introduction/third-party-libraries/camunda-bpm-rpa-bridge-license-book.md:1]: :x: The sentence `For more information,  including possible other licensing options, contact FasterXML.com (http://fasterxml.com).` should be removed for the rest of the jackson libraries. As discussed in https://github.com/camunda/camunda-docs-manual/pull/1428#discussion_r1147387091

**Review by yanavasileva — COMMENTED** [review]
👍 Platform changes look good. rpa bridge page needs adjustments.

File: content/introduction/third-party-libraries/camunda-bpm-rpa-bridge-license-book.md
Comment (ThorbenLindhauer) [reply content/introduction/third-party-libraries/camunda-bpm-rpa-bridge-license-book.md:1]: Update via https://github.com/camunda/camunda-docs-manual/pull/1427/commits/b8465e9194468dd1aa41d38f43704787cd6c61b7

**Review by yanavasileva — APPROVED** [review]

**Pull request merged**